tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -311,6 +311,13 @@ _ when x.ToBoolean() is { } boolean => boolean,
                                       .WithKeys(ConfigurationKeys.ApmTracingEnabled)
                                       .AsBool(defaultValue: true);
 
+            var ciVisibilityEnabled = config
+                                     .WithKeys(ConfigurationKeys.CIVisibility.Enabled)
+                                     .AsBool(defaultValue: false);
+
+            // If CI Visibility is enabled then we should enable tracing.
+            _traceEnabled = _traceEnabled || ciVisibilityEnabled;
```

Comment:
What about `DD_APM_TRACING_ENABLED` 🤔  Should that be updated?

Reply:
oh, I didn't know anything about that env-var, I would say probably yes.

Reply:
done in [d2f83c3](https://github.com/DataDog/dd-trace-dotnet/pull/6951/commits/d2f83c3557a437e62f2fa1763f5c68bcd6ade89a)